test/fixtures.ts

```diff
@@ -532,6 +532,7 @@ export const allFixtures: Fixture[] = [
   new JSONFixture(languages.CPlusPlusLanguage),
   new JSONFixture(languages.ElmLanguage),
   new JSONFixture(languages.SwiftLanguage),
+  new JSONFixture(languages.SwiftDenseLanguage),
```

Comment:
We have the `quickTestRendererOptions` which only test a subset of the tests for non-default options you can specify, which is probably what you want here.